wasm/jsapi/functions/entry.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Entry settings object for host functions</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/wasm/jsapi/wasm-module-builder.js"></script>
+<script src="/wasm/jsapi/function/helper.js"></script>
+<!-- This is the entry page, so window.open() should resolve relative to it, even inside host functions. -->
+
+<iframe src="resources/entry-incumbent.html"></iframe>
+
+<script>
+function call_later(f) {
+  const builder = new WasmModuleBuilder();
+  const functionIndex = builder.addImport("module", "imported", kSig_v_v);
```

Comment:
Optional style idea: I personally find it clearer if references to cross-file global properties are prefixed with `window`. E.g. I am assuming `kSig_v_v` and `functionIndex` are defined in one of the `/wasm/` scripts, and it would be clearer if they were written as `window.kSig_v_v` and `window.functionIndex`. (This feels less necessary for constructors like `WasmModuleBuilder`, but I could go either way.)

Reply:
`functionIndex` is defined on this line. `window.kSig_v_v` doesn't work because it's defined with `let`.